Sources/Testing/Traits/Trait.swift

```diff
@@ -12,75 +12,79 @@
-/// if it can be added to test functions, ``SuiteTrait`` if it can be added to
-/// test suites, and both ``TestTrait`` and ``SuiteTrait`` if it can be added to
-/// both test functions _and_ test suites.
+/// - term ``TestTrait``: Conform to this type in traits that you add to test
+///   functions.
+/// - term ``SuiteTrait``: Conform to this type in traits that you add to test
+///   suites.
+///
+/// You can add a trait that conforms to both ``TestTrait`` and ``SuiteTrait``
+/// to test functions and test suites.
 public protocol Trait: Sendable {
-  /// Prepare to run the test to which this trait was added.
+  /// Prepare to run the test that has this trait.
   ///
   /// - Parameters:
-  ///   - test: The test to which this trait was added.
+  ///   - test: The test that has this trait.
   ///
-  /// - Throws: Any error that would prevent the test from running. If an error
-  ///   is thrown from this method, the test will be skipped and the error will
-  ///   be recorded as an ``Issue``.
+  /// - Throws: Any error that prevents the test from running. If an error
+  ///   is thrown from this method, the test is skipped and the error is
+  ///   recorded as an ``Issue``.
   ///
-  /// This method is called after all tests and their traits have been
-  /// discovered by the testing library, but before any test has begun running.
-  /// It may be used to prepare necessary internal state, or to influence
+  /// The testing library calls this method after it discovers all tests and
+  /// their traits, and before it begins to run any tests.
+  /// Use this method to prepare necessary internal state, or to determine
   /// whether the test should run.
   ///
   /// The default implementation of this method does nothing.
   func prepare(for test: Test) async throws
 
-  /// The user-provided comments for this trait, if any.
+  /// The user-provided comments for this trait.
   ///
-  /// By default, the value of this property is an empty array.
+  /// The default value of this property is an empty array.
   var comments: [Comment] { get }
 
   /// The type of the test scope provider for this trait.
   ///
-  /// The default type is `Never`, which cannot be instantiated. The
-  /// ``scopeProvider(for:testCase:)-cjmg`` method for any trait with this
-  /// default type must return `nil`, meaning that trait will not provide a
-  /// custom scope for the tests it's applied to.
+  /// The default type is `Never`, which can't be instantiated. The
+  /// ``scopeProvider(for:testCase:)-cjmg`` method for any trait with
+  /// `Never` as its test scope provider type must return `nil`, meaning that
+  /// the trait doesn't provide a custom scope for tests it's applied to.
   associatedtype TestScopeProvider: TestScoping = Never
 
-  /// Get this trait's scope provider for the specified test and/or test case,
-  /// if any.
+  /// Get this trait's scope provider for the specified test or test case.
```

Comment:
The "and/or" was there because, there is _always_ a `Test` passed, it's just the `Test.Case` parameter which is optional. Any ideas for a better way to phrase that? Saying "test **or** test case" makes it sound like it's one or the other.

Reply:
Right, good point. I avoided and/or because it's difficult for readers of English as a second language. I've gone with "test, and optional test case". I'm not sure I'm entirely happy with that but maybe it helps advance the discussion :).